cluster/images/kubemark/BUILD

```diff
@@ -12,8 +12,8 @@ docker_build(
 docker_push(
     name = "push",
     image = ":image",
-    registry = "gcr.io",
-    repository = "$(PROJECT)/kubemark",
+    registry = "$(REGISTRY)",
+    repository = "kubemark",
```

Comment:
@ixdy @BenTheElder Does this look okay from the bazel side?

Reply:
This looks like valid bazel, seems fine to me.